src/Common/src/Interop/Unix/System.Net.Http.Native/Interop.Easy.cs

```diff
@@ -124,6 +139,19 @@ internal enum CurlProtocols
             CURLPROTO_HTTPS = (1 << 1),
         }
 
+        // Enum for constants defined for the results of CURL_SEEKFUNCTION
+        internal enum CurlSeekResult : int
+        {
+            CURL_SEEKFUNC_OK = 0,
+            CURL_SEEKFUNC_FAIL = 1,
+            CURL_SEEKFUNC_CANTSEEK = 2,
+        }
+
+        // constants defined for the results of a CURL_READ or CURL_WRITE function
+        internal const ulong CURL_READFUNC_ABORT = 0x10000000;
```

Comment:
The reasoning is that the places where these are used don't return a "code" or "enum". Instead the method is supposed to return the "number of bytes transferred". These values are "magic values" to signify abort/pause. (I don't know what happens when you actually want to return that exact number of bytes, but this is how libcurl is defined.)
So the method's signature is to return a size_t in libcurl (ulong in our managed shim).
So instead of making these enums, and then casting their values to ulong in the consuming code, I thought it was more appropriate to just make these ulong constants.